pages/_app.tsx

```diff
@@ -73,6 +74,8 @@ const SmallBusinessHeroApp = ({Component, pageProps}) => {
 
 	const classes = useStyles();
 
+	initializeAnalytics();
```

Comment:
This code runs each time the component is rendered. Currently, the root component is not likely to rerender, but we will not know when that changes. You can move this line outside the `SmallBusinessHeroApp` component to make sure it executes only once.

Reply:
Resolved in  0e01fea